vignettes/why-it-works.Rmd

```diff
@@ -0,0 +1,285 @@
+
+$$
+\Delta_{w}f(t) = f(t + w) - f(t).
+$$
+
+### Uniform primary event time ($r=0$)
+
+Applying a uniform primary event time distribution to equation \@ref(eq:survivalfunc) gives:
+
+$$
+Q_{S_+}(t) = Q_T(t + w_P) + { 1 \over w_P} \int_0^{w_P} f_T(t+p) p~ dp.
+$$
+This is analytically solvable whenever the upper distribution function of $T$ is known and the mean of $T$ is analytically solvable from its integral definition.
+
+In each case considered below it is easier to change the integration variable:
+
+$$
+Q_{S_+}(t) = Q_T(t + w_P) + { 1 \over w_P} \int_t^{t+w_P} f_T(z) (z-t)~ dz = Q_T(t + w_P) + { 1 \over w_P} \Big[  \int_t^{t+w_P} f_T(z) z~ dz - t \Delta_{w_P}F_T(t) \Big]. (\#eq:unifprim)
+$$
+
+**General partial expectation**
+
+Note that for any distribution with an analytically available distribution function $F_T$ equation \@ref{eq:unifprim} can be solved so long as the _partial expectation_
+
+$$
+\int_t^{t+w_P} f_T(z) z~ dz (\#eq:partexp)
+$$
+
+can be reduced to an analytic expression.
+
+**General Discrete censored delay distribution**
+
+First we note that equation \@ref{eq:disccensprob} can be written using the difference operator: $f_n = -\Delta_1 Q_{S_+}(n-1)$. We can insert this expression into equation \@ref{eq:unifprim} to give the discrete censored delay distribution for uniformly distributed primary event times:
+
+$$
+f_n = \Delta_1\Big[(n-1) \Delta_1F_T(n-1)\Big] - \Delta_1Q_T(n) + \Delta_1\Big[ \int_{n-1}^n f_T(z) z ~dz \Big] = (n+1)F_T(n+1)  + (n-1)F_T(n-1) - 2nF_T(n) + \Delta_1\Big[ \int_{n-1}^n f_T(z) z ~dz \Big]. (\#eq:disccensunifprim)
+$$
+
+We now consider some specific delay distributions.
+
+#### Gamma distributed delay times
+
+The Gamma distribution has the density function:
+
+$$
+f_T(z;k, \theta) = {1 \over \Gamma(k) \theta^k} z^{k-1} \exp(-z/\theta).
+$$
+Where $\Gamma$ is the Gamma function.
+
+The Gamma distribution has the distribution function:
+
+$$
+\begin{aligned}
+F_T(z;k, \theta) &= {\gamma(k, z/\theta) \over \Gamma(k)}, \qquad z\geq 0,\\
+F_T(z;k, \theta) &= 0, \qquad z < 0.
+\end{aligned}
+$$
+Where $\gamma$ is the lower incomplete gamma function.
+
+**Gamma partial expectation**
+
+$$
+\begin{aligned}
+\int_t^{t+w_P} f_T(z) z~ dz &= {1 \over \Gamma(k) \theta^k} \int_t^{t+w_P}z z^{k-1} \exp(-z/\theta)~dz \\
+&=  {\Gamma(k+1) \theta^{k+1} \over \Gamma(k) \theta^k}  {1 \over \Gamma(k+1) \theta^{k+1}} \int_t^{t+w_P}z^{k} \exp(-z/\theta)~dz\\
+&= k\theta \Delta_{w_P} F_T(t; k + 1, \theta).
+\end{aligned} (\#eq:gammapartexp)
+$$
+
+**Survival function of $S_{+}$ for Gamma distribution**
+
+By substituting equation \@ref{eq:gammapartexp} into equation \@ref{eq:disccensunifprim} we can solve for both the survival function of $S_+$ in terms of analytically available functions:
+
+$$
+\begin{aligned}
+Q_{S_+}(t; k, \theta) &= Q_T(t + w_P; k, \theta) + { 1 \over w_P} \left( {1 \over \Gamma(k) \theta^k} \int_t^{t+w_P} z^{k-1}  \exp(-z/\theta) (z-t)~ dz \right), \\
```

Comment:
do you think we actually need the first line or is by subsitutation and the second line just enough? In some sense it feels like scary maths that isn't actually adding anything now?